tools/scripts/validate_alerts.py

```diff
@@ -0,0 +1,39 @@
+import argparse
+import json
+import jsonschema
```

Comment:
Just curious if you know an equivalent library to `jsonschema` for YAML.  If there is one, it could be an option to consider given that we are using mostly YAML config in PyTorch CI https://github.com/pytorch/pytorch/tree/main/.github

Reply:
this is a possibility https://github.com/keleshev/schema